modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -423,7 +423,7 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
          *  @param outputBlobs contains all output blobs for each layer specified in @p outBlobNames.
          *  @param outBlobNames names for layers which outputs are needed to get
          */
-        void forward(std::vector<std::vector<Mat> >& outputBlobs,
+        void forward(CV_OUT std::vector<std::vector<Mat> >& outputBlobs,
```

Comment:
We've removed `CV_WRAP` macro here because it's wrapped similarly to a previous one method. So there are two ways  to do `forward` pass in python: with a single string or with a list of strings. May be it's better to wrap it by `CV_WRAP_AS` using different name?